### PR TITLE
nf core template merge 2.6

### PR DESCRIPTION
**Pull request by maxulysse**
Replace #772
To merge after sync with #771 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules.json [hunk 1]
```diff
@@ -144,8 +144,7 @@
                     },
                     "gatk4/cnnscorevariants": {
```
Comment (asp8200) [inline modules.json:145]: Should there also be an update to the sha/version of the multiqc-module?

File: modules/nf-core/multiqc/main.nf [hunk 2]
```diff
@@ -1,14 +1,16 @@
 process MULTIQC {
```
Comment (asp8200) [inline modules/nf-core/multiqc/main.nf:1]: Doesn't these changes to `modules/nf-core/multiqc/main.nf` mean that the sha of `multiqc` in `modules.json` should be updated?

File: workflows/sarek.nf [hunk 3]
```diff
@@ -1050,15 +1048,25 @@ workflow SAREK {
...
 
-        MULTIQC(ch_multiqc_files.collect(), ch_multiqc_configs.collect())
+        MULTIQC (
+            ch_multiqc_files.collect(),
+            ch_multiqc_config.collect().ifEmpty([]),
+            ch_multiqc_custom_config.collect().ifEmpty([]),
+            ch_multiqc_logo.collect().ifEmpty([])
+        )
         multiqc_report = MULTIQC.out.report.toList()
+        ch_versions    = ch_versions.mix(MULTIQC.out.versions)
```
Comment (asp8200) [inline workflows/sarek.nf:1069]: hmm ... Is the channel `ch_versions` used after that point in the code?

File: modules/nf-core/multiqc/main.nf (hunk 2 above)
Comment (maxulysse) [reply modules/nf-core/multiqc/main.nf:1]: sha has been updated actually

File: modules.json (hunk 1 above)
Comment (maxulysse) [reply modules.json:145]: done, I actually removed the patch I was using before

File: workflows/sarek.nf (hunk 3 above)
Comment (maxulysse) [reply workflows/sarek.nf:1069]: not that I'm aware of

Comment (asp8200) [reply workflows/sarek.nf:1069]: hmm ... if it isn't used then why is line 1069 there?

Comment (maxulysse) [reply workflows/sarek.nf:1069]: it was in TEMPLATE, but I agree, don't make much sense.
I actually have some ideas for that later on, but for now, we can keep it

Comment (asp8200) [reply workflows/sarek.nf:1069]: ok

**Review by asp8200 — APPROVED** [review]
seems legit

**Pull request merged**